testing/xfpga/test_sysfs_c.cpp

```diff
@@ -179,6 +179,7 @@ TEST_P(sysfsinit_c_p, sysfs_initialize) {
         devs.erase(id);
       }
     }
+    return FPGA_OK;
```

Comment:
What are the advantages of having a return for this CB func?

Reply:
This CB is a trivial one but I changed the signature of the callback to return an fpga_result as an indication in the loop of the `sysfs_foreach_device` function to abort the loop and return the result code.